### ISSUE TITLE
Fix the 2 v18 accessibility bugs 1873 and 1874 in the Digital Design repo

### ISSUE THREAD

**Issue by RobertaJHahn** [carbon-design-system/carbon-for-ibm-dotcom#5155]
#### User Story

> As a `[user role below]`:
DDS engineer

> I need to:
fix the accessibility bugs for Drupal

> so that I can:
support the RFP between IBM and the State

#### Additional information

- v18 bugs to address and include in r148a 
-[Accessibility (ACR) violation on Search box, located at the top right of any IBM web page](https://zenhub.ibm.com/app/workspaces/digital-design-system-5b3a815a8bf6a932efa6fd03/issues/webstandards/digital-design/1873)
 -[The script element has the id "utag_24_0" that is already in use.](https://zenhub.ibm.com/app/workspaces/digital-design-system-5b3a815a8bf6a932efa6fd03/issues/webstandards/digital-design/1874)
 - **These issues must be fixed on the web site by March 1st.  Additional instructions are included in the bug**
 - Schedule to fix
 -DDS development: Feb 15 - Feb 22
 -Notify issue reporter that bugs are ready for testing & instructions to test:  Feb 22
 -Address any bugs found by adopter:  Feb 23 - 26

#### Acceptance criteria
 - [ ] Fix provided to adopter and tested by the adopter
 - [ ] r148a is made available
